src/main/webapp/themeAndWidgetsFragment.xhtml

```diff
@@ -180,7 +181,7 @@
                     </ui:fragment>
                 </p:fragment>
                 <div class="button-block">
-                    <p:commandButton class="btn btn-default" value="#{bundle.saveChanges}" update=":#{p:component('messagePanel')}, :themeWidgetsForm" action="#{themeWidgetFragment.save()}" oncomplete="bind_bsui_components();"/>
+                    <p:commandButton class="btn btn-default" value="#{bundle.saveChanges}" update=":#{p:resolveClientId('messagePanel', view)}, :themeWidgetsForm" action="#{themeWidgetFragment.save()}" oncomplete="bind_bsui_components();"/>
```

Comment:
https://github.com/primefaces/primefaces/wiki/Migration-Guide#others-1 says that p:component has been deprecated but as of 81d6837 I still see it below on line 253:

```

253:    <p:remoteCommand name="saveRedirectUrl" process="@this" update=":#{p:component('messagePanel')}, :themeWidgetsForm, @([id$=Messages])" actionListener="#{themeWidgetFragment.save()}"/>
````

Reply:
Fixed.